### DISCUSSION TITLE
Auto-Delete Sandbox

### DISCUSSION THREAD

**Discussion by MickesGer** [sandboxie-plus/Sandboxie#4434] [General]
Ich habe unter Windows 11 bei mehreren PC's das Problem, dass die Sandbox beim Beenden nicht gelöscht wird und eine Fehlermeldung ausgegeben wird. Ich habe bisher auch keine Lösung finden können, so dass ich es jetzt anders gelöst habe. Vielleicht ist dies auch für Andere hilfreich.

1. Erstelle eine .bat-Datei, z.B. "SB_Ende.bat"
2. Schreibe in die Bat-Datei "rd /s /q C:\Sandbox\Home\DefaultBox"
3. Lege die Datei z.B. im Sandbox-Ordner ab
4. Öffne die Sandbox-Einstellungen und gehe unter "Erweiterte Optionen" auf den Reiter "Auslöser".
5. Wähle rechts unter den Optionen bei "Beim Box Beenden" den Punkt "Kommando ausführen"
6. Gebe dort nun den Pfad zur Bat-Datei ein, z.B. "c:\Sandbox\SB_Ende.bat"
7. Fertig

So, ab jetzt wird die Sandbox ohne Probleme beim Beenden der Box gelöscht

--------------------------

English via Google:

I have the problem on several PCs under Windows 11 that the sandbox is not deleted when closing and an error message is displayed. I have not been able to find a solution so far, so I have solved it differently now. Maybe this will be helpful for others too.

1. Create a .bat file, e.g. "SB_Ende.bat"
2. Write "rd /s /q C:\Sandbox\Home\DefaultBox" in the bat file
3. Save the file in the sandbox folder, for example
4. Open the sandbox settings and go to the "Trigger" tab under "Advanced options".
5. On the right under the options under "When box closes" select "Execute command"
6. Now enter the path to the bat file there, e.g. "c:\Sandbox\SB_Ende.bat"
7. Done

So, from now on the sandbox will be deleted without any problems when the box is closed

**Comment by offhub** [comment]
[1.15.4 is released](https://github.com/sandboxie-plus/Sandboxie/releases/latest)
